main/lsp/json_types.cc

```diff
@@ -88,4 +88,61 @@ string JSONBaseType::toJSON() const {
     return buffer.GetString();
 }
 
+// Object-specific helpers
+
+unique_ptr<Range> Range::fromLoc(const core::GlobalState &gs, core::Loc loc) {
+    if (!loc.exists()) {
+        // this will happen if e.g. we disable the stdlib (e.g. to speed up testing in fuzzers).
+        return nullptr;
```

Comment:
this predates us introducing usage of `optional` but this probably should be an optional now